<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newItemDialog</class>
 <widget class="QDialog" name="newItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelItemType">
      <property name="text">
       <string>Item Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelItemName">
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBoxItemType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItemName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddItem">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDone">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
